packages/zoe/src/internal-types.js

```diff
@@ -231,11 +234,17 @@
  * ability to call terminate(), Zoe makes this visible.
  *
  * @property {() => Promise<void>} done
- * provides a promise that will be fulfilled when the contract is terminated.
- * @property {() => void} terminate
- * kills the vat in which the contract is running
+ * returns a promise that will be fulfilled or rejected when the contract is
+ * shutdown.
+ * @property {(reason: string) => void} terminate
+ * Terminate the vat in which the contract is running. This bypasses
+ * notification of Zoe. zcf.shutdown('reason') is much more graceful.
+ * @property {(reason: string) => void} terminateOnFailure
+ * Terminate the vat in which the contract is running and indicate failure. This
+ * bypasses notification of Zoe. zcf.shutdownOnFailure('reason') is much more
+ * graceful.
```

Comment:
inconsistent language here, shutdown vs terminate.

Reply:
It wasn't inconsistent, as zcf still has `shutdown()`.  But there's no reason to compare to shutdown since this is no longer visible to contracts.